domain/event.go

```diff
@@ -0,0 +1,34 @@
+package domain
+
+import (
+	"time"
+
+	"github.com/gofrs/uuid"
+)
+
+// Event knoQ上のイベント情報
+type Event struct {
+	ID          uuid.UUID
+	Name        string
+	Description string
+	GroupID     uuid.UUID
+	RoomID      uuid.UUID
+	TimeStart   time.Time
+	TimeEnd     time.Time
+	SharedRoom  bool
+	Level       EventLevel
+}
+
+// EventLevel 0 匿名で公開, 1 全て公開, 2 部内にのみ公開
```

Comment:
```suggestion
// EventLevel
```
それぞれのことは下のconstのそれぞれの行にコメントつけてほしい